packages/material-ui/src/internal/SelectableGroup/SelectableGroupContext.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+
+/**
+ * @ignore - internal component.
+ */
+const SelectableGroupContext = React.createContext();
```

Comment:
How would you handle something like Radio where it can either have a provider or be standalone? Currently, there's a check to see if the context is null to work which state it is in.

Reply:
Sounds like we allow single Radio buttons? That doesn't sound that useful. How should the radio button behave without context?

Reply:
https://5c89a4198f06b5000805780e--material-ui.netlify.com/demos/selection-controls/#standalone-radio-buttons

Reply:
So they're either controlled without context or uncontrolled with context? Seems useful. I guess this does make sense after all.

Reply:
Yeah, I could always add a value like `providerPresent` to be passed down?

Reply:
I think the current approach is good enough. If the amount of `context && context.value` guards is way more common than only checking for `context` we can think about a better strategy.